### PR TITLE
Profile and minor refactoring

### PR DESCRIPTION
**Pull request by samyakshah3008**
Profile Feat add and minor refactoring done

Live link: https://triples-chat-69878d.netlify.app/

### CODE REVIEW

**Review by surbhi84 — APPROVED** [review]
Code looks good, naming could be improved.
Do check for unnecessary comments.
The feature is working fine.
Looks good to merge.

**Review by ranitasahaa1230 — APPROVED** [review]
The code looks clean and simple
The folder structure is proper
The feature is working properly and the UI is so cool. Good job
Looks good to merge

File: src/components/FollowUserChip.jsx [hunk 1]
```diff
@@ -25,29 +28,38 @@ export default function FollowUserChip() {
     loadData();
   }, []);
 
-  const currentUser = users?.find(
-    (each) => each?.username === userInfo?.username
+  // const currentUser = users?.find(
+  //   (each) => each?.username === userInfo?.username
+  // );
```
Comment (Developers-codz) [inline src/components/FollowUserChip.jsx:33]: can remove commented code

**Review by Developers-codz — APPROVED** [review]
The code and Folder structure both looks good
UI also looks good, give some right margin to use listing component and also remove commented code else look good to merge. Good Job

**Pull request merged**